plugins/com.google.cloud.tools.eclipse.appengine.facets/plugin.xml

```diff
@@ -46,4 +46,44 @@
     </listener>
   </extension>
 
+  <extension point="org.eclipse.ui.commands">
+    <command
+      description="%convertToAppEngineStandardCommandDescription"
+      id="com.google.cloud.tools.eclipse.appengine.facets.convertToStandard"
+      name="%convertToAppEngineStandardCommandName">
+    </command>
+  </extension>
+
+  <extension point="org.eclipse.ui.menus">
+    <menuContribution
+      allPopups="false"
+      locationURI="popup:org.eclipse.ui.popup.any?after=additions">
+      <command
+        commandId="com.google.cloud.tools.eclipse.appengine.facets.convertToStandard"
+        label="%convertToAppEngineStandardMenuLabel">
+        <visibleWhen checkEnabled="true" />
+      </command>
+    </menuContribution>
+  </extension>
+
+  <extension point="org.eclipse.ui.handlers">
+    <handler
+      class="com.google.cloud.tools.eclipse.appengine.facets.ui.AppEngineStandardProjectConverter"
+      commandId="com.google.cloud.tools.eclipse.appengine.facets.convertToStandard">
+      <enabledWhen>
+        <and>
+          <count value="1" />  <!-- enabled when only one project is selected -->
+          <iterate>
+            <adapt type="org.eclipse.core.resources.IProject" />
+          </iterate>
+          <not>
+            <iterate>
+              <reference definitionId="com.google.cloud.tools.eclipse.appengine.onlyInGCPProjects" />
```

Comment:
@briandealwis This test is negated by `<not>`. The name of the test is also misleading in that it tests (before the negation) if the project has the App Engine Standard facet.

That said, this will show the conversion menu for *any* project that doesn't have the Standard facet. Now I wonder if we need to narrow down the scope, e.g., by checking if the project has a directory `WEB-INF`.